### PR TITLE
JS: Allows setting query named and default graphs

### PR DESCRIPTION
**Pull request by Tpt**

### CODE REVIEW

File: js/src/store.rs [hunk 1]
```diff
@@ -50,6 +50,8 @@ export class Store {
         options?: {
             base_iri?: NamedNode | string;
             results_format?: string;
+            default_graph?: BlankNode | DefaultGraph | NamedNode | Iterable<BlankNode | DefaultGraph | NamedNode>;
+            named_graphs?: Iterable<BlankNode | NamedNode>;
```
Comment (tpluscode) [inline js/src/store.rs:54]: Curious, how do blank nodes make sense as query context?

Comment (tpluscode) [reply js/src/store.rs:54]: Also, nitpick, I'd propose to keep a consistent form. In SPARQL protocol both are singular
```suggestion
            default_graph?: BlankNode | DefaultGraph | NamedNode | Iterable<BlankNode | DefaultGraph | NamedNode>;
            named_graph?: Iterable<BlankNode | NamedNode>;
```

Comment (Tpt) [reply js/src/store.rs:54]: Here Oxigraph considers blank node as terms in the queried dataset, opposite to the query string where blank nodes are assumed to be variables. This provides a way to query specifically a graph identified by a blank node.

For example if `default_graph: [blankNode("a)]` the query will use the named graph `_:a` as the default graph.

Comment (Tpt) [reply js/src/store.rs:54]: >Also, nitpick, I'd propose to keep a consistent form and type. In SPARQL protocol both are singular.

Good point. The Rust and Python APIs are plural (`named_graphs`). Hence, I used plural for consistency. These parameters are not exactly the same as the SPARQL protocol (they allows blank nodes and the default graph).

Comment (tpluscode) [reply js/src/store.rs:54]: Right. I don't think I've seen blank node graph identifiers outside of n3 rules 🤔

Comment (Tpt) [reply js/src/store.rs:54]: Yes, it's quite rare but supported by RDF 1.1

**Pull request merged**